templates/registration/totp_enable.html

```diff
@@ -42,10 +42,15 @@
             image-rendering: -moz-crisp-edges;
             image-rendering: crisp-edges;
         }
+
+        .scratch-codes {
+            white-space: pre-line;
+        }
     </style>
 {% endblock %}
 
-{% block js_media %}
+{% block content_js_media %}
+    {{ super() }}
```

Comment:
Isn't `common-content.html`'s `content_js_media` empty? (and also I don't think any other templates that extend from `common-content.html` have a similar `super()` call)